Lesson2/Chapter4/Zad7.py

```diff
@@ -0,0 +1,29 @@
+'''
+Created on 23 lut 2018
+
+@author: agata.wiewiora
+
+Given N numbers: the first number in the input is N, 
+after that N integers are given. 
+Count the number of zeros among the given integers and print it.
+
+You need to count the number of numbers 
+that are equal to zero, not the number of zero digits. 
+
+'''
+
+N = int(input('N '))
```

Comment:
wielkimi literami w pythonie zapisuje się stałe (zgodnie z PEP8)

Reply:
To jest zgodne z opisem zad, zatem zostawię w tym przypadku